src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -340,14 +340,15 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
       case None => Put
     }
 
-    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } }
-
-    // don't leak any sensitive data
-    RequestComplete(NoContent)
-  }.recover {
+    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } } map { response =>
```

Comment:
In this case it's DB queries, but there's also a chance that we'll be hammering Google too if there's a huge change in the whitelist. For example, there are about 30 controlled access workspaces that every dbGapAuthorizedUser is given access to. If the whitelist updates and we see that we need to remove 50 users, we'll end up making 30x50 calls to Google to remove each individual from each workspace's read intersection group. I haven't yet seen a way to batch update Google groups :(